jonprl-mode.el

```diff
@@ -558,7 +558,21 @@ Invokes `jonprl-mode-hook'."
   ;; This is a customizable hook, hence the indirection.
   (add-hook 'after-save-hook 'jonprl-mode-run-after-save-hook t t)
   (yas-define-snippets 'jonprl-mode jonprl-vernacular-snippets)
-  (jonprl-update-operators))
+  (jonprl-update-operators)
+
+  (eval-after-load 'flycheck
+    '(progn
+       (flycheck-define-checker jonprl
+                                "A JonPRL proof checker."
```

Comment:
I can definitely believe the indentation is weird 😉 

This is what happened when I auto-indented, but maybe I should have put linebreaks in a different place?